index.js

```diff
@@ -3656,6 +3656,43 @@
       [Fn(a, b), m(a), m(a)],
       Z.sortBy);
 
+  //# zipWith :: (a -> b -> c) -> Array a -> Array b -> Array c
+  //.
+  //. Creates a new array out of the two supplied arrays
+  //. by applying the given function to each equally-positioned
+  //. pair of elements from each given array. The returning
+  //. array is truncated to the length of the shorter of the two
+  //. input arrays.
```

Comment:
This could be shortened a little. I suggest:

_Returns the result of combining, pairwise, the given arrays using the given binary function. The length of the resulting array is equal to the length of the shorter input array._